src/test/kotlin/ar/edu/unahur/obj2/semillasAlViento/ParcelaTest.kt

```diff
@@ -0,0 +1,74 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.booleans.shouldBeFalse
+import io.kotest.matchers.shouldBe
+
+class ParcelaTest : DescribeSpec({
+    val parcela1 = Parcela(20, 1, 9)
+    val parcela2 = Parcela(20, 1, 9)
+
+
+    val soja1 = Soja(2009, 1.5f)
+    //no da semillas porque es transgenica
+    val soja2 = SojaTransgenica(2009, 1.5f)
+
+
+    val agricultora1 = Agricultora(parcelas = listOf(parcela1, parcela2))
+
+
+
+    describe("Parcela1") {
+
+        it("test") {
+            parcela1.superficie().shouldBe(20)
+            parcela1.cantidadMaximaPlantas().shouldBe(4)
+        }
+        it("plantar una planta") {
+            parcela1.plantar(soja1)
+            parcela1.plantas.size.shouldBe(1)
+
+        }
+        it("tiene Complicaciones") {
+            parcela1.plantar(soja1)
+            parcela1.plantas.size.shouldBe(1)
+            parcela1.tieneComplicaciones().shouldBeFalse()
+
+
+        }
+
+
+    }
+    describe("agricultora") {
+
+
+        it("es semillera") {
+            parcela1.plantar(soja1)
+            //soja2 no da semillas porque es transgenica
+            parcela2.plantar(soja2)
+            agricultora1.parcelasSemilleras().count().shouldBe(1)
```

Comment:
Mejor mirar qué elementos tiene la lista:
```suggestion
            agricultora1.parcelasSemilleras().shouldContainExactly(soja1)
```